Common/UnitDefinitions/Power.json

```diff
@@ -94,6 +94,19 @@
           "Abbreviations": [ "Btu/hr" ]
         }
       ]
+    },
+    {
+      "SingularName": "SolarLuminosity",
```

Comment:
Should this perhaps go in `Luminosity.json` ? Or is it more correct as `Power`?

Reply:
https://en.wikipedia.org/wiki/Luminosity

I guess there is some overlap in units (Watt, joules per second), but it does seem like Luminosity is considered a separate type of quantity since they define it as `electromagnetic power`. What do you think?

Reply:
So, what do you decide on this? Keep it in Power or add a new Luminosity quantity?